docs/blog/2018-10-09-vscode-gatsby-development/index.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Using VS Code for Supercharged Gatsby.js Development"
+author: Michael Holtzman
+date: 2018-10-09
+tags: ["editor", "source", "documentation"]
+---
+
+The Gatsby.js platform and ecosystem is composed not only of the [core Inkteam members](https://www.gatsbyjs.com/about/), but a huge group of community members as well. I have no doubt that part of the success of Gatbsy is due to the core team really encouraging the community to be involved. They even give you [free swag for becoming a contributor](/docs/contributor-swag/).
+
+Now that I have convinced you to get involved, the first thing you should do is review the existing [how to contribute guide](/docs/how-to-contribute/). That will go over the basics of checking out the project and getting set up.
+
+The remainder of the article will talk about using VS Code if you want to contribute to the core [Gatsby project](https://github.com/gatsbyjs/gatsby). That being said everything mentioned here can be applied to just about any JavaScript project.
+
+## Editors - Make Your Selection
+
+![Lots of Available Options](./images/color-wall.jpeg)
+
+I know that there are as many choices for IDE editors and personal preferences as there are JS frameworks in existence (hint LOTS). I am not going to debate those here, this article will focus on [VS Code](https://code.visualstudio.com/). It's the editor I use and find it amazing for the Javascript and web development ecosystem. I was a long time holdout using Sublime Text, but once I gave Code a try I switched over within a day and haven't looked back.
```

Comment:
Probably be consistent. If you call it `VS Code` in one place, call it that in all places, e.g. lose `Code`!

Reply:
(this is not a strong opinion, just a suggestion!)